runPyRelValThread.py

```diff
@@ -197,6 +198,7 @@ def parseLog(self):
         logFile = logData[wf]['steps'][step]
         json_cache = os.path.dirname(logFile)+"/logcache_"+str(step)+".json"
         if (not os.path.exists(json_cache)) or (os.path.getmtime(logFile)>os.path.getmtime(json_cache)):
+          log_payload(logFile)
```

Comment:
add try catch block here so that runPyRelValThread does not fail due to error in new code